### ISSUE TITLE
DAC 22 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 09

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#22]
<img width="377" alt="DAC_Colour_Contrast_Non_Text_09" src="https://user-images.githubusercontent.com/75850106/155562170-c14d4773-98c4-4ae8-9fba-94b7925758fb.png">

### Issue
Lines between places on the maps are faint and fail to meet the minimum colour ratio of 3:1, meaning that low vision users may have a difficult experience seeing where places begin and end.\

Foreground: #B4B5B5 
Background: #F2F3F0
The contrast ratio is: 1.8:1 
1.4.11 Non-text Contrast (AA)

<img width="604" alt="DAC 22 Current code ref" src="https://user-images.githubusercontent.com/75850106/155562313-4900d7d1-11f8-4f34-b4c3-1bb9d0236205.png">

### Suggested Solution
Ensure that the lines that split up places on the map are obvious to low vision users. The colour contrast of the lines must be a minimum of 3:1 against adjacent colours.
